OpenRA.Mods.Common/Traits/World/TeamTogether.cs

```diff
@@ -0,0 +1,63 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
```

Comment:
2022

Reply:
[4aaad09](https://github.com/OpenRA/OpenRA/pull/19144/commits/4aaad09ba6a325aa053324f8c9148ac100a66ecf) should resolve this